vue-vtkjs-pvw-template/src/components/core/App/template.html

```diff
@@ -12,11 +12,16 @@
       :min="3"
       :step="1"
     />
-    <v-switch :class="$style.switch" hide-details v-model="darkMode" />
-  </v-toolbar>
+    <v-switch
+      :class="$style.switch"
+      @click="$vuetify.theme.dark = !$vuetify.theme.dark"
+    />
+  </v-app-bar>
 
-  <v-content :class="$style.appContent">
-    <remote-rendering-view :client="client" />
-    <!-- <vtk-view ref="vtkViewComponent"/> -->
-  </v-content>
+  <v-main :class="$style.appContent">
+    <div style="position: relative; width: 100%; height: 100%">
+      <remote-rendering-view :client="client" />
+      <!-- <vtk-view ref="vtkViewComponent"/> -->
```

Comment:
delete that line

Reply:
(Delete this whole component, it confused you before!)